std/bigint.d

```diff
@@ -277,8 +277,8 @@ public:
 
     // BigInt op integer
     BigInt opBinary(string op, T)(T y) pure nothrow const
-        if ((op=="+" || op == "*" || op=="-" || op=="/" || op=="|" || op=="&" ||
-            op=="^"|| op==">>" || op=="<<" || op=="^^")
+        if ((op=="+" || op == "*" || op=="-" || op=="/" || op=="%" || op=="|" ||
+            op=="&" || op=="^" || op==">>" || op=="<<" || op=="^^")
```

Comment:
Hmm, your idea keeps consistency!
The below looks nice.

- `BigInt % long` => `long`
- `BigInt % ulong` => `BigInt`